app/controllers/concerns/authenticatable.rb

```diff
@@ -61,11 +61,14 @@ def require_logged_in_admin
     end
   end
 
-  def require_logged_in_user_or_400
+  def require_logged_in_user_or_401
```

Comment:
While the status's name "Unauthorized" is more-specific, the RFC defines it _even more_ specifically and [requires that it set another header](https://www.rfc-editor.org/rfc/rfc7235#section-3.1) to lead into the HTTP basic auth or digest auth flow, neither of which Lobsters supports.

So 401 is not appropriate here. I could [see a case for 403](https://www.rfc-editor.org/rfc/rfc7231#section-6.5.3), I don't think there's anything wrong there. Our login page is long enough that browsers shouldn't preempt it with their builtin pages.